tools/cbmc/proofs/HTTP/IotHttpsClient_ReadResponseStatus/Makefile.json

```diff
@@ -1,5 +1,10 @@
 {
   "ENTRY": "IotHttpsClient_ReadResponseStatus",
+  
+  # This is the length of the user data after the header in a buffer.
+  # This can be set to any size.
+  "USER_DATA_SIZE": "1000",
```

Comment:
Same comments as above: suggest 3 different configurations, suggest not defining a variable here